src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -138,62 +138,44 @@
   no_log: true
   when: not fake_ssl_cert and commtrack_key_value is defined
 
-- name: Copy icds tableau SSL cert
+- name: Link CAS SSL cert
   become: yes
-  copy:
-    content: "{{ tableau_nginx_combined_cert_value }}"
-    dest: "{{ ssl_certs_dir }}/{{ tableau_nginx_ssl_cert }}"
-    mode: 0400
-    owner: root
-    group: root
-  when: not fake_ssl_cert and tableau_nginx_combined_cert_value is defined
-
-- name: Copy icds tableau SSL Key
-  become: yes
-  copy:
-    content: "{{ tableau_key_value }}"
-    dest: "{{ ssl_keys_dir }}/{{ tableau_nginx_ssl_key }}"
-    mode: 0400
-    owner: root
-    group: root
-  no_log: true
-  when: not fake_ssl_cert and tableau_key_value is defined
-
-- name: Copy CAS SSL cert
-  become: yes
-  copy:
-    content: "{{ cas_nginx_combined_cert_value }}"
+  file:
+    src: "/etc/letsencrypt/live/{{CAS_SITE_HOST}}/fullchain.pem"
     dest: "{{ ssl_certs_dir }}/{{ cas_nginx_ssl_cert }}"
-    mode: 0400
-    owner: root
-    group: root
-  when: not fake_ssl_cert and cas_nginx_combined_cert_value is defined
+    state: link
+  when: not fake_ssl_cert and CAS_SITE_HOST | default(None) and letsencrypt_cchq_ssl
```

Comment:
why the need to check `letsencrypt_cchq_ssl`

Reply:
Thanks for catching it Simon. That defenetly makes sense not to use it. 
There is still some work left in this PR. Sorry I did not marked it [WIP]